api/src/main/java/edu/cornell/mannlib/vitro/webapp/config/ConfigurationProperties.java

```diff
@@ -26,98 +25,69 @@
 public abstract class ConfigurationProperties {
 	private static final Log log = LogFactory
 			.getLog(ConfigurationProperties.class);
-
-	/** The bean is attached to the session by this name. */
-	private static final String ATTRIBUTE_NAME = ConfigurationProperties.class
-			.getName();
-
+	
 	/** If they ask for a bean before one has been set, they get this. */
 	private static final ConfigurationProperties DUMMY_PROPERTIES = new DummyConfigurationProperties();
 
+    private static ConfigurationProperties INSTANCE = DUMMY_PROPERTIES;
+
+    public static ConfigurationProperties getInstance() {
+        if (INSTANCE == DUMMY_PROPERTIES) {
+            log.error("ConfigurationProperties bean has not been set.");
+        }
+        return INSTANCE;
```

Comment:
Do we need DUMMY_PROPERTIES anymore? Can we check whether INSTANCE == null instead?